src/renderer/scss/component/_header.scss

```diff
@@ -4,39 +4,144 @@
 
   align-items: center;
   background-color: $lbry-white;
-  box-shadow: 0 1px 5px rgba($lbry-black, 0.1);
   display: flex;
   justify-content: space-between;
-  padding: 0 $spacing-width * 1/3;
   position: fixed;
   z-index: 1;
 
-  @media (min-width: $medium-breakpoint) {
-    padding: 0 $spacing-width;
+  html[data-theme='dark'] & {
+    background-color: rgba($lbry-black, 0.9);
+    color: $lbry-white;
   }
 }
 
-.header__actions-right {
+.header__navigation {
   display: flex;
-  margin-left: auto;
 
-  .btn {
-    margin-left: $spacing-width * 1/3;
+  &:first-of-type {
+    html[data-theme='dark'] & {
+      border-color: $lbry-gray-5;
+    }
+
+    @media (min-width: 601px) {
```

Comment:
Should these be variables? If it's specific to header stuff maybe just add a comment because it's confusing from a glance.

Reply:
If 600px is a global width that will trigger ui changes, it should be a variable. But if only the header is changing at 600px then a comment explaining that will help.

Reply:
Ah gotcha. I'll add comments to all media queries then.